src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1052,7 +1060,7 @@ public new NpgsqlDataReader ExecuteReader(CommandBehavior behavior = CommandBeha
             => ExecuteReaderAsync(behavior, async: false, CancellationToken.None).GetAwaiter().GetResult();
 
         /// <summary>
-        /// An asynchronous version of <see cref="ExecuteReader"/>, which executes
+        /// An asynchronous version of <see cref="ExecuteReader()"/>, which executes
```

Comment:
This change was because the overload was ambiguous between the no-parameter and default-parameter members of the same name.